alexthompson207/whats-cookin
# Feature/refactor dom ids and amounts display

## Is this a fix or a feature?

This is a fix.

## What is the change?

See below.

## What does it fix?

Refactor querySelectors to getElementById, target HTML ids instead of class names.
Refactor `calculateRecipeCost()` method, to display 2 decimal places.
Refactor `returnIngredientAmounts()` method to round ingredient amounts of a recipe to 1 decimal place.

## Where should the reviewer start?

In `scripts.js` file lines 13-29.
In `recipe.js` file:
lines 28-37, `returnIngredientAmounts()` method
lines 39-46, `calculateRecipeCost()` method

## How should this be tested?

Click on a recipe in locally hosted website. Recipe total cost should only have 2 decimal places, and recipe ingredient amounts should only have 1 decimal place.

## Screenshots (if appropriate):

![screencapture-file-Users-alexthompson-turing-mod-2-projects-whats-cookin-src-index-html-2021-02-16-15_54_35](https://user-images.githubusercontent.com/71950194/108131969-55e9d280-706f-11eb-8ddf-21b3a59f3b7f.png)
